README.md

```diff
@@ -1 +1,110 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+## Problem
+
+This is a challenge by Trustly known as the "Parliament Challenge".
+
+The Swedish parliament has an open API platform for developers to use its resources to create applications. Among its resources are the speeches in the parliament and the data of its members.
+
+The challenge by Trustly is to create an appliction that serves requests for parliament speeches. The expected response should be created by merging the data from the speeches api and members api and should be in a JSON format.
+
+The expected response should at least contain:
+
+- A unique ID, representing the speech itself (anforande_id)
+- Date of speech (dok_datum)
+- The name of the speaker, and only the name (tilltalsnamn)
+- Political affiliation (parti)
+- The official e-mail address (uppgift)
+- Constituency (valkrets)
+- A decent sized image that could be used in a web site listing (bild_url_192)
+- The debate subject (avsnittsrubrik)
+- A link to the speech (HATEOAS style) (protokoll_url_www)
```

Comment:
For each field in the bracket, also indicate which api it's coming from, i.e. members api or speeches api.

Reply:
added fields